<?xml version="1.0"?>

<launch>

  <node pkg="volti" type="imu_node" name="imu_plate" respawn="true">
    <rosparam>
        rate: 20
        topic: imu_plate_3
        i2c: 1
  	</rosparam>
  </node>

  <node pkg="volti" type="imu_node" name="imu_pendu" respawn="true">
    <rosparam>
        rate: 20
        topic: imu_pendu_3
        i2c: 2
    </rosparam>
  </node>

  <node pkg="volti" type="double_motor_adjust.py" name="double_motor_adjust" respawn="true">
  </node>

  <node pkg="volti" type="push_servos.py" name="push_servos" respawn="true">
  </node>

  <node pkg="volti" type="readencs.py" name="readencs" respawn="true">
  </node>

  <node pkg="volti" type="control_iface.py" name="control_iface" respawn="true">
  </node>

</launch>
